swift.en.md

```diff
@@ -0,0 +1,340 @@
+# Swift coding style
+
+## Introduction
+
+This document defines the conventions for writing Swift code at Cookpad Inc.
+
+This guide directed for Swift 3.0.2.
+
+## Objective
+
+This guides aim to accomplish these objectives.
+
+1. Clarify purposes 
+1. Improve readability
+1. Keep consistency
+1. Reduce side-effects
+
+## Code style
+
+- **[MUST]** Use four spaces for 1-level of indent.  Do not use the horizontal tab character.
+- **[MUST]** Add linebreak to end of files
+- **[MUST]** Place open brackets on same lines
+
+```swift
+// Bad
+func update() 
+{
+}
+
+// Good
+func update() {
+}
+```
+
+- **[MUST]** Use trailing comma on elements of collections
+    - Because of reducing file diffs
+
+```swift
+// Bad
+let ingredients = [
+    "bacon": "100g",
+    "lettuce": "1/4",
+    "tomato": "half"
+]
+
+// Good
+let ingredients = [
+    "bacon": "100g",
+    "lettuce": "1/4",
+    "tomato": "half",
+]
+```
+
+- **[MUST]** Do not use `()` wraps condition statements
+
+```swift
+// Bad
+if (a == 0) {
+}
+
+// Good
+if a == 0 {
+}
+```
+
+- **[MUST]** Do not use `;` on end of lines
+
+## Class, Struct
+
+- **[MUST]** Do not inherit `NSObject` except if needed to refer from Objective-C.
+- **[SHOULD]** Do not use `mutating` attribute for structs.
+    - Because structs should be immutable.
```

Comment:
It seems that it make sense to split this part in two sentences but for English maybe we can combine it into one like:
`Since structs should be immutable, do not use `mutating` attribute for structs.`